language/move-lang/src/typing/globals.rs

```diff
@@ -173,6 +177,13 @@ fn exp_list_item(
     }
 }
 
+fn is_current_function(context: &Context, call: &T::ModuleCall) -> bool {
+    match (&context.current_module, &context.current_function) {
+        (Some(m), Some(f)) => m == &call.module && f == &call.name,
```

Comment:
Do you need to worry about module aliases here? Or are the `current_module` and the `call` only un-aliased module names?

Reply:
Aliasing is removed two compiler passes earlier 
expansion > naming > typing
So no need to worry